### DISCUSSION TITLE
Configuring open-webui in Windows Server 2019

### DISCUSSION THREAD

**Discussion by Micahelb** [open-webui/open-webui#1609] [General]
# Bug Report
Error when running the commands.
docker run -d -p 3000:8080 --add-host=host.docker.internal:host-gateway -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main

docker pull ghcr.io/open-webui/open-webui:main

## Description
Being trying to deploy open-webui on my Windows Server 2019 VM, which already has OLLAMA running fine with LLAMA2 module.
I've been following the official guithub and webpage of Open Web UI.

**Bug Summary:**
I have OLLAMA running fine as I checked by browsing http://127.0.0.1:11434.
I am getting error when I ran this command, 
docker run -d -p 3000:8080 --add-host=host.docker.internal:host-gateway -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main

ERROR:
PS C:\Ollama-web\open-webui> docker run -d -p 3000:8080 --add-host=host.docker.internal:host-gateway -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main
Unable to find image 'ghcr.io/open-webui/open-webui:main' locally
main: Pulling from open-webui/open-webui
docker: no matching manifest for windows/amd64 10.0.17763 in the manifest list entries.
See 'docker run --help'.

As per documentation here: https://docs.openwebui.com/getting-started/#quick-start-with-docker-, as long as I have OLLAMA up and running and so docker, this command should work.

Upon further investigation, I tried to run a docker pull against the supposed image repo, and seems that on this repo there isn't an image matching Windows Server 2019 built.

ERROR2:
PS C:\Ollama-web\open-webui> docker pull ghcr.io/open-webui/open-webui:main
main: Pulling from open-webui/open-webui
no matching manifest for windows/amd64 10.0.17763 in the manifest list entries

**Steps to Reproduce:**
Downloaded and Installed OLLAMA from here: https://ollama.com/download/windows
Downloaded and Installed Docker on Windows Server as per MS article: https://learn.microsoft.com/en-us/virtualization/windowscontainers/quick-start/set-up-environment?tabs=dockerce#windows-server-1
Downloaded and Installed Git from here: https://git-scm.com/download/win
Cloned Git with command: git clone https://github.com/open-webui/open-webui.git

## Environment

- **Operating System:** [Windows Server 2019 Datacenter]

## Reproduction Details

**Confirmation:**

- [X] I have read and followed all the instructions provided in the README.md.
- [X] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

**Comment by metromancn** [comment]
Same error, I am using Windows Server 2022 on Cloud Server. The CPU is Intel Xeon. BTW, since Docker Desktop can't be installed on Cloud Windows Server, I am using Docker CE for instead.

**Comment by Al3x888** [comment]
i am having this same problem. Is there any work arounds
